packages/grid/_modules_/grid/utils/utils.ts

```diff
@@ -82,3 +84,7 @@ export function mapColDefTypeToInputType(type: string) {
 
 // Util to make specific interface properties optional
 export type Optional<T, K extends keyof T> = Pick<Partial<T>, K> & Omit<T, K>;
+
+export function getButtonBaseUtilityClass(slot) {
+  return generateUtilityClass(GRID_CSS_CLASS_PREFIX, slot);
```

Comment:
what is this one? What issue does it solve?

Reply:
This one is used to generate the class names. We didn't complete the migration, we are not supposed to have `GRID_HEADER_CELL_CSS_CLASS` variables like with the approach of v5 in the core. Instead, we would read the global gridClasses.headerCell.